Makefile

```diff
@@ -141,19 +130,16 @@ bench: fmt
 test:
 	go test -short -tags='debug testing netgo' -timeout=5s $(pkgs) -run=. -count=$(count)
 
-test-long: lint lint-ci start-mongo
+test-long: lint lint-ci start-mongo test-long-ci stop-mongo
+
+test-long-ci:
 	@mkdir -p cover
-	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=60s $(pkgs) -run=$(run) -count=$(count)
-	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=600s $(integration-pkgs) -run=$(run) -count=$(count)
-	-make stop-mongo
-
-# test-single allows us to run a single integration test.
-# Make sure to start MongoDB yourself!
-# Example: make test-single RUN=TestHandlers
-test-single: export COOKIE_HASH_KEY="7eb32cfab5014d14394648dae1cf4e606727eee2267f6a50213cd842e61c5bce"
-test-single: export COOKIE_ENC_KEY="65d31d12b80fc57df16d84c02a9bb62e2bc3b633388b05e49ef8abfdf0d35cf3"
-test-single:
-	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=300s $(integration-pkgs) -run=$(run) -count=$(count)
+	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=600s $(pkgs) -run=$(run) -count=$(count)
+
+# Cookie vars
+# TODO: Are these used?
```

Comment:
Not sure how or why these were being used for the `test-single`, but also `test-single` isn't needed, just a `run` parameter needed to be defined at the beginning of the Makefile in order for updating it at run time.